docpages/building/linux.md

```diff
@@ -36,12 +37,27 @@ Once installed to the `/usr/local` directory, you can make use of the library in
 g++ -std=c++17 mydppbot.cpp -o dppbot -ldpp
 ```
 
+If you are on Arch Linux you can add `/usr/local` to your enviorment variable:
```

Comment:
Spelling mistake and punctuation mistake here too.

Correct to `If you are on Arch Linux, you will need to add `/usr/local` to your environment variables:`